mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -260,7 +260,7 @@ def Rock_GridwiseGemmV2Op :
                    // Note: num_cu isn't used
                    I32Attr:$blockSize,
                    I32Attr:$gridSize,
-                   Rock_XdlopsGemmParamsAttr:$params)> {
+                   OptionalAttr<XdlopsGemmParams>:$params)> {
```

Comment:
By the time we get to gridwise gemm, the tuning parameters are no longer optional. Please don't mark them as such.

This goes for any of the gridwise_, blockwise_, and so on.